src/main.tsx

```diff
@@ -1,14 +1,10 @@
-import React from 'react'
-import ReactDOM from 'react-dom'
+import { createRoot } from 'react-dom/client'
 import App from './App'
 
 import '@unocss/reset/tailwind.css'
 import './styles/main.css'
 import 'uno.css'
 
-ReactDOM.render(
-  <React.StrictMode>
```

Comment:
Maybe we can keep the `StrictMode`.

https://github.com/vitejs/vite/blob/main/packages/create-vite/template-react/src/main.jsx
https://reactjs.org/blog/2022/03/29/react-v18.html